webpack.common.config.ts

```diff
@@ -618,6 +618,20 @@ module.exports = {
         }
       ]
     },
+    {
+      test: /\.(jpe?g|png|gif|svg)$/i,
+      include: [
+        path.resolve(__dirname, 'assets'),
+        path.resolve(__dirname, 'core/templates'),
+        path.resolve(__dirname, 'extensions'),
+        path.resolve(__dirname, 'typings'),
+        path.resolve(__dirname, 'node_modules'),
+      ],
+      loader: 'file-loader',
+      options: {
+        name: '/guppy-dev/build/icons/help.png'
```

Comment:
@shivkant5 I have a question. WIll this config ask webpack to go through asssets, core/templates, extensions, typings and the entire node modules to find all image (jpg, jpeg, png, gif, svg) files and have them available as an import called /guppy-dev/build/icons/help.png? If yes, that would be overkill for just a help icon